control/bdalg.py

```diff
@@ -60,14 +60,16 @@
 
 __all__ = ['series', 'parallel', 'negate', 'feedback', 'append', 'connect']
 
-def series(sys1, sys2):
-    """Return the series connection sys2 * sys1 for --> sys1 --> sys2 -->.
+def series(sys1, sys2,*sysn):
+    """Return the series connection (... * sys3 *) sys2 * sys1 
```

Comment:
How about
```
def series(sys1, *sysn):
    from functools import reduce
    return reduce(lambda x, y: y*x, sysn, sys1)
```
and similarly for parallel?

One downfall there is that `series(s1)` would return the `s1` instance, and not a copy of it.

Reply:
Depending on one's experience, the use of `reduce` and `lambda` is elegant. Given the simplicity of the operation, the more important question here is performance. Sometimes functional idioms like the one suggested by @roryyorke can be better optimized by the Python compiler than a for-loop.

Reply:
+1 for `reduce` and `lambda`